src/components/layout/title/Layout.tsx

```
@@ -118,6 +118,19 @@ export default function TitleLayout() {
       };
       break;
 
+    case pathname.startsWith('/schedule/') && pathname.endsWith('/participation'):
+      // URL에서 type 파라미터 확인
+      const urlParams = new URLSearchParams(window.location.search);
+      const type = urlParams.get('type');
+      headerProps = {
+        isBack: true,
+        isTitle: true,
+        titleText: type === 'settlement' ? '정산 현황' : '참여 현황',
+        isLike: false,
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)